File: Plus - minus - plus - plus - ... - Count/Plus - minus - plus - plus - ... - Count.dart
int catchSignChange (List<int> arr) {  
  int count = 0;
  for(int i=0;i<arr.length-1;i++){
    if((arr[i]>=0 && arr[i+1]<0) || (arr[i]<0 && arr[i+1]>=0)){
      count+=1;
    }
  }
  return count;
}

File: Plus - minus - plus - plus - ... - Count/Plus - minus - plus - plus - ... - Count.js
function catchSignChange(arr) {
  let count=0;
  for(let i=0;i<arr.length-1;i++){
    if((arr[i+1]>=0 && arr[i]<0) || (arr[i+1]<0 && arr[i]>=0)){
      count+=1
    }
  }
  return count;
}
